docs/network_area_diagrams/index.md

```diff
@@ -0,0 +1,46 @@
+# Network area diagrams
+
+The powsybl-network-area-diagram artifact provides features to generate concise and customized diagrams of the network.
+
+```{toctree}
+---
+maxdepth: 2
+---
+
+layout/layout.md
+```
+
+## Introduction
```

Comment:
Can't we remove this line? I'm not very fond of adding an introduction title. What's the sentence before the first title then, if not an introduction itself?